modules/admin_manual/pages/configuration/server/activity_configuration.adoc

```diff
@@ -21,7 +21,11 @@ If it is not enabled, go to your ownCloud Apps page to enable it.
 
 == Configuring your ownCloud for the Activity App
 
+[TIP] 
+====
 A working e-mail configuration is required to configure your ownCloud to send out e-mail notifications.
 Furthermore, it is recommended to configure the xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[background job] `Webcron` or `Cron`.
+====
 
+Email notifications for shared files can be enabled/disabled by administrators with the "_Allow users to send mail notifications for shared files to other users_", available in menu:Settings[Admin > Sharing].
```

Comment:
This has nothing to do with the Activity app. The setting is to allow/disallow sending public links via mail from within the web interface.